tests/kafkatest/tests/core/downgrade_test.py

```diff
@@ -139,13 +139,13 @@ def test_upgrade_and_downgrade(self, version, compression_types, security_protoc
 
         self.logger.info("First pass bounce - rolling upgrade")
         self.upgrade_from(kafka_version)
-        self.run_validation()
```

Comment:
Hmm, perhaps we could assert something about producer/consumer progress just to be sure we have a non-trivial downgrade scenario?